### PR TITLE
[cmake] Cosmetics: core_find_versions

### PR DESCRIPTION
**Pull request by hudokkow**
## Description
~~Set mirrors URL from version.txt~~

* ~~First commit adds `APP_MIRRORS` support.~~
* ~~Second~~ commit is (mostly) cosmetics.
* ~~Third commit renames `KODI_MIRROR` to `APP_MIRRORS` under `project`, `tools/buildsteps` and `tools/depends`. Did it for consistency sake and it **wasn't tested**. That's what jenkins is for... ;)~~

There are still some places with hardcoded stuff. Something for another day.

## Motivation and Context
Been hanging around on one of my branches for over a year. Time to PR.

## How Has This Been Tested?
Built on Ubuntu 17.10.

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: version.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 APP_NAME Kodi
 COMPANY_NAME XBMC Foundation
 WEBSITE http://kodi.tv
+MIRRORS http://mirrors.kodi.tv
```
Comment (Rechi) [inline version.txt]: I don't like to set this it here.
This file is to control the produced artefacts (exe, deb, dmg, apk) for all platforms, not to control compile settings which do not have any effect on the build output.

Comment (wsnipex) [reply version.txt]: agreed, this doesn't belong here.
And please keep KODI_MIRROR as alias, if APP_MIRROR is not defined.

File: cmake/scripts/common/Macros.cmake [hunk 2]
```diff
@@ -761,6 +762,9 @@ macro(core_find_versions)
   # unset variables not used anywhere else
   unset(version_list)
   unset(APP_APP_NAME)
+  unset(APP_COMPANY_NAME)
```
Comment (Rechi) [inline cmake/scripts/common/Macros.cmake]: https://github.com/xbmc/xbmc/blob/6c4ba80b694141d83968d66a15607c4e7d367de5/tools/windows/packaging/uwp/package.appxmanifest.in#L14

**Comment by hudokkow** [comment]
Fixed, thanks!

**Comment by hudokkow** [comment]
jenkins build this please

**Review by wsnipex — APPROVED** [review]

**Pull request merged**